lang/de/lang.php

```diff
@@ -1,24 +1,56 @@
 <?php
-
 /**
  * @license    GPL 2 (http://www.gnu.org/licenses/gpl.html)
- * 
+ *
  */
+
+// banner
 $lang['apr_recent_draft']      = 'Die <a href="%s">aktuellste Version</a> dieses Dokuments ist ein Entwurf.';
 $lang['apr_outdated']          = 'Diese Version wurde durch eine <a href="%s">neuere bestätigte Version</a> <em>ersetzt</em>.';
 $lang['apr_draft']             = 'Diese Version (%s) ist ein <em>Entwurf</em>.';
 $lang['apr_approved']          = 'Diese Version (%s) wurde <em>bestätigt</em> durch %s.';
 $lang['apr_previous']          = 'Die <a href="%s">zuvor bestätigte Version</a> (%s) ist verfügbar.';
+
+// Pending Approvals List
 $lang['apr_p_none']            = 'Es gibt keine Dokumente die bestätigt werden müssen';
 $lang['apr_p_hdr_page']        = 'Seite';
 $lang['apr_p_hdr_previous']    = 'Zuletzt bestätigt';
 $lang['apr_p_hdr_updated']     = 'Aktualisiert';
 $lang['apr_p_approved']        = 'Von %s am <a href="%s">%s</a>';
+
+// Submit (Next to the tick-box to approve a page)
 $lang['apr_do_approve']        = 'Bestätigen';
+
+
 $lang['wrong permissions to approve'] = 'Sie haben nicht die nötige Berechtigung um diese Seite zu bestätigen.';
 $lang['cannot approve a non-existing revision'] = 'Sie können keine Revision bestätigen die nicht existiert.';
 $lang['version approved']      = 'Sie haben das Dokument überprüft.';
 $lang['cannot approve error']  = 'Überprüfung konnte aus technischen Gründen nicht gesetzt werden. (Berechtigungsfehler?)';
+
 $lang['approvals']             = 'Überprüfungen: %s/%s';
 $lang['approved by']           = 'Überprüft durch: %s';
 $lang['approve action']        = 'Dokument bestätigen';
+
+$lang['mail_greeting'] = 'Hallo @FULLNAME@!';
+
+$lang['mail_new_suggestiopns'] = 'Ein neuer Vorschlag für @TITLE@ auf @DOKUWIKIURL@ wurde erstellt.
+
+Ansehen and Bestätigen: @URL@';
+
+$lang['mail_changes_to_approved_rev'] = 'Änderungen zur letzten bestätigten Version: @CHANGES@';
+$lang['mail_changes_to_previous_rev'] = 'Änderungen zur vorherigen Version: @CHANGES@';
+
+$lang['mail_approved'] = 'Ihr Vorschlag für @TITLE@ auf @DOKUWIKIURL@ wurde bestätigt.
+
+URL: @URL@';
+
+
+
+$lang['mail_dw_signature'] =
```

Comment:
the signature should be part of the mail body until splitbrain/dokuwiki/pull/1101 is merged